go/vt/sqlparser/comments.go

```diff
@@ -32,6 +32,8 @@ const (
 	DirectiveQueryTimeout = "QUERY_TIMEOUT_MS"
 	// DirectiveScatterErrorsAsWarnings enables partial success scatter select queries
 	DirectiveScatterErrorsAsWarnings = "SCATTER_ERRORS_AS_WARNINGS"
+	// DirectiveMaxPayloadSizeOverride skips payload size validation when set.
+	DirectiveMaxPayloadSizeOverride = "MAX_PAYLOAD_SIZE_OVERRIDE"
```

Comment:
Shouldn't this be called `IGNORE_MAX_PAYLOAD_SIZE`? `OVERRIDE` makes me think that you are going to provide a different value for `maxPayloadSize`

Reply:
Updated this to be `IGNORE_MAX_PAYLOAD_SIZE`